doc/waarp-r66/source/configuration-details/server.rst

```diff
@@ -86,11 +86,53 @@ Section ``network``
 Balise             Type    Obl. Défaut    Signification
 ================== ======= ==== ========= =============
 serverport         integer N    6666      Port utilisé pour le protocole R66
+serverips          string  N    null      Adresses utilisées pour le protocole R66 (séparées par des virgules)
 serversslport      integer N    6667      Port utilisé pour le protocole R66 en SSL
+serversslips       string  N    null      Adresses utilisées pour le protocole R66 en SSL (séparées par des virgules)
 serverhttpport     integer N    8066      Port utilisé pour l'interface web de supervision (la valeur ``0`` désactive l'interface)
+serverhttpips      string  N    null      Adresses utilisées pour l'interface web de supervision (séparées par des virgules)
 serverhttpsport    integer N    8067      Port utilisé pour l'interface web HTTPS d'administration (la valeur ``0`` désactive l'interface)
+serverhttpsips     string  N    null      Adresses utilisées pour l'interface web HTTPS d'administration (séparées par des virgules)
 ================== ======= ==== ========= =============
 
+.. versionadded:: 3.4.1
```

Comment:
This change makes the next version 3.5.0 instead of 3.4.1

Reply:
Agree (while it does not change a lot, it is a new functionality)